company-profiles/bharatpe.md.txt

```diff
@@ -0,0 +1,14 @@
+# BharatPe
+
+## Company
+
+BharatPe is a leading Fintech BharatPe was founded in 2018 with the vision of making financial inclusion a reality for Indian merchants. In 2018, BharatPe launched India’s first UPI interoperable QR code, the first zero MDR payment acceptance service. In 2020, post-Covid, BharatPe also launched India’s only zero MDR card acceptance terminals – BharatSwipe. Currently serving over 80 lakh merchants across 225+ cities , the company is a leader in UPI offline transactions, processing 18 crores+ UPI transactions per month (annualized Transaction Processed Value of US$ 18.5 Bn in annualised payments). The company has already facilitated disbursement of loans totalling to over ₹7000 crores to over 400,000 merchants, since launch. BharatPe’s POS business processes payments of over US$ 4 bn annually on its POS machines. It has a network of 1.25 lac+ machines across cities. BharatPe has raised over US$ 650 million in equity and debt, till date. The company’s list of marquee investors includes Tiger Global, Dragoneer Investment Group, Steadfast Capital, Coatue Management, Ribbit Capital, Insight Partners, Steadview Capital, Beenext, Amplo and Sequoia Capital.  In June 2021, the company announced the acquisition of PAYBACK India, the country’s largest multi-brand loyalty program company with 100 million+ members. In October 2021, the consortium of Centrum Financial Services Limited (Centrum) and BharatPe, was issued a Small Finance Bank (SFB) license by the Reserve Bank of India (RBI). BharatPe also made its entry in the Buy Now Pay Later segment with the launch of postpe in October 2021.
+
+## Company size
+
+0 - 1000
+	
+## Remote status
+
+Bharatpe works to provide a professional, hybrid , fun environment at its Gurgaon Office, including healthy snacks; an open, inclusive workspace onsite fitness and trainings for all employees, flexible work schedules.
+
```

Comment:
- This file should just be called `company-profiles/bharatpe.md` without the `.txt.`
- There some missing sections, were they missed deliberately?